vm/src/stdlib/os.rs

```diff
@@ -2466,6 +2466,32 @@ mod posix {
         })
     }
 
+    extern "C" {
+        fn ctermid(s: *mut libc::c_char) -> *mut libc::c_char;
+    }
+    #[pyfunction]
+    fn _ctermid(vm: &VirtualMachine) -> PyResult {
+        let returned = unsafe { ctermid(std::ptr::null_mut()) };
+        if returned.is_null() {
+            Err(errno_err(vm))
+        } else {
+            let returned = unsafe { ffi::CStr::from_ptr(returned) }.to_str().unwrap();
+            Ok(vm.ctx.new_str(returned))
+        }
+    }
+
+    #[pyfunction]
+    // fchdir is the same as os.chdir(file descriptor)
+    // 1. have a file desriptor that repsresents a directory
+    // 2. pass this fd to the fchdir function
+    // 3. check if the fd is a directory or not
+    // 4. if its a dirrectory, then proceed to chage current directory to the fd's path
```

Comment:
Some small typos in the comment: desriptor -> descriptor. repsresents -> represents. dirrectory -> directory.

Is this supposed to be a doc comment? then it should have three slashes right?

Reply:
no - its just some pseudo code / my thinking strategy of implementing fchdir - i will remove it once i work through the implementation

Reply:
Ah ignore my comment then, thought that it was meant to stay ;)

Reply:
haha nooooo :P thanks for your (future) review. looking forward to it :)